infra/feast-operator/internal/controller/services/services.go

```diff
@@ -352,6 +352,7 @@ func (feast *FeastServices) setPod(podSpec *corev1.PodSpec) error {
 	feast.mountTlsConfigs(podSpec)
 	feast.mountPvcConfigs(podSpec)
 	feast.mountEmptyDirVolumes(podSpec)
+	feast.mountPodVolumes(podSpec)
```

Comment:
thoughts? something else?
```suggestion
	feast.mountServiceVolumes(podSpec)
```